casepro/pods/registry.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+import importlib
+
+
+def get_class_from_string(string):
+    '''
+    Given a class specified by a string, this will return that class.
+    '''
+    module_name, cls_name = string.rsplit('.', 1)
+    module = importlib.import_module(module_name)
+    return getattr(module, cls_name)
+
+
+def load_pod(index, config):
+    '''
+    Given the index of the pod, and the config dictionary for the pod, this
+    returns the instance of that pod.
+    '''
+    class_name = config.pop('type')
+    config['index'] = index
+    return get_class_from_string(class_name)(config)
```

Comment:
It would be nice to be able to leverage `django.apps.get_app_configs()` to load the pods like we did in the test branch, would avoid the need to be loading classes from strings ourselves and would mean we can use `INSTALLED_APPS` to register pod types. That will allow us to include each pod's static files in the directories django looks for static files in without needing to add any boilerplate.